autorag/indexer/process/azure/paragraph.py

```diff
@@ -0,0 +1,111 @@
+from llama_index import Document
+from llama_index.schema import TextNode
+from llama_index.node_parser import SentenceSplitter
+
+# Define roles to be excluded
+DEFAULT_EXCLUDED_ROLES: list[str] = ["pageHeader", "pageNumber"]
+
+# Define contents to be excluded
+DEFAULT_EXCLUDED_CONTENTS: list[str] = ["Contains Nonbinding Recommendations"]
```

Comment:
is it application specific?

Reply:
DEFAULT_EXCLUDED_ROLES and DEFAULT_EXCLUDED_CONTENTS are content-related only to the paragraph processor, and they do not need to change for every model config. Do you think it would be better to keep them in this file?